src/test/java/org/kainos/ea/db/AuthRoleDaoTest.java

```diff
@@ -0,0 +1,120 @@
+package org.kainos.ea.db;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.cli.AuthRole;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+
+
+@ExtendWith(MockitoExtension.class)
+public class AuthRoleDaoTest {
+    private DatabaseConnector databaseConnector = Mockito.mock(DatabaseConnector.class);
+    private Connection connection = Mockito.mock(Connection.class);
+    private PreparedStatement statement = Mockito.mock(PreparedStatement.class);
+    private ResultSet resultSet = Mockito.mock(ResultSet.class);
+
+
+    private AuthRoleDao authRoleDao = new AuthRoleDao();
```

Comment:
For correctness you should recreate the class in each test to make sure tests can't affect each other. You can do this in a function that runs before the test